ui/src/layout/Menu.js

```diff
@@ -23,23 +25,43 @@ const translatedResourceName = (resource, translate) =>
         : inflection.humanize(inflection.pluralize(resource.name)),
   })
 
+const useStyles = makeStyles({
+  menuWrapper: {
+    maxHeight: '100%',
+    display: 'flex',
+    flexDirection: 'column',
+  },
+})
+
 const Menu = ({ onMenuClick, dense, logout }) => {
   const isXsmall = useMediaQuery((theme) => theme.breakpoints.down('xs'))
   const open = useSelector((state) => state.admin.ui.sidebarOpen)
+  const [playlistMenuName, setPlaylistMenuName] = useState('')
   const translate = useTranslate()
   const resources = useSelector(getResources)
 
   // TODO State is not persisted in mobile when you close the sidebar menu. Move to redux?
   const [state, setState] = useState({
     menuAlbumList: true,
     menuLibrary: true,
+    menuPlaylists: true,
     menuSettings: false,
   })
 
   const handleToggle = (menu) => {
     setState((state) => ({ ...state, [menu]: !state[menu] }))
   }
 
+  useEffect(() => {
+    if (resources?.length) {
+      const playListResource = resources.find(
+        (resource) => resource.name === 'playlist'
+      )
+      setPlaylistMenuName(translatedResourceName(playListResource, translate))
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
The solution here is to add `translate` to the dependencies below, not to ignore the lint rule. Or am I missing something?

Reply:
Oh, it's just that I couldn't see why a function should be added to a dependency list.